### DISCUSSION TITLE
1,000,000 downloads!

### DISCUSSION THREAD

**Discussion by sfirke** [sfirke/janitor#437] [Show and tell]
Hi janitor friends -

I wrote a blog post reflecting on the milestone and the journey: https://samfirke.com/2021/03/13/reflections-on-the-janitor-r-package/ :tada: 

I figure sharing it here is the best way to reach you, all of whom I've recognized in the post either as individuals or as a community (if I left someone out by name, sorry!).  The community around this Github repo has made the package possible :pray:  Thanks everyone and congrats!

Sam

**Comment by billdenney** [comment]
Thank you so much for the kind words!  I am happy to have contributed to your project and that it has been so useful to so many people!

**Comment by jzadra** [comment]
First off, congratulations! What a milestone.  The blog post was a great read.  I admit I am one who never looked into how adorn_* does its magic!  Thanks for mention as well, I hope to keep contributing!
